### PR TITLE
Player: Fix empty img tag occupy 20px size in safari. v6.0.142

### PR DESCRIPTION
**Pull request by suzp1984**
the html img tag occupy 20px size in safari. 
https://github.com/ossrs/srs/blob/427104f1dab86f5afc7d7b49b02ed27d03ef9346/trunk/research/players/rtc_player.html#L19

> Screenshot 2024-04-17 at 9 17 07 AM

(ignore the img css warning: `auto\9;` it's another problem, I will file another PR.)

but, the empty img tag just occupy 1px size in chrome. So I guess it's a html compatible problem.

> Screenshot 2024-04-17 at 9 46 33 AM

---------

`TRANS_BY_GPT4`

### CODE REVIEW

**Comment by suzp1984** [comment]
I guess the `//ossrs.net/gif/v1/sls.gif?site=ossrs.net&path=/console/enindex` is used for statistics & analysis purpose. but pls pay attention to the path, for some one may not so match with its real path.

e.g.
https://github.com/ossrs/srs/blob/427104f1dab86f5afc7d7b49b02ed27d03ef9346/trunk/research/console/ng_index.html#L23
https://github.com/ossrs/srs/blob/427104f1dab86f5afc7d7b49b02ed27d03ef9346/trunk/research/players/srs_gb28181.html#L31
https://github.com/ossrs/srs/blob/427104f1dab86f5afc7d7b49b02ed27d03ef9346/trunk/research/players/whep.html#L19
https://github.com/ossrs/srs/blob/427104f1dab86f5afc7d7b49b02ed27d03ef9346/trunk/research/players/whip.html#L19

File: trunk/research/console/en_index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <script type="text/javascript" src="js/srs.console.js"></script>
 </head>
 <body ng-controller="CSCMain">
-<img src='//ossrs.net/gif/v1/sls.gif?site=ossrs.net&path=/console/enindex'/>
+<img style="width: 1px; height: 1px;" src='//ossrs.net/gif/v1/sls.gif?site=ossrs.net&path=/console/enindex'/>
```
Comment (winlinvip) [inline trunk/research/console/en_index.html]: Why not change to:

```css
width: 0px; height: 0px;
```

Comment (suzp1984) [reply trunk/research/console/en_index.html]: `with: 0px; height: 0px;` also works, but chrome will render this `img` tag in `1px` size by default, so I did this to keep consistency with chrome.

Comment (suzp1984) [reply trunk/research/console/en_index.html]: change to `width: 0px; height: 0px`, and confirmed the `gif` request can still send: `http://ossrs.net/gif/v1/sls.gif?site=ossrs.net&path=/player/srsplayer`.

**Pull request merged**